blaze/compute/sql.py

```diff
@@ -111,6 +111,16 @@ def compute_up(t, s, **kwargs):
     return compute(t._scalar_expr, d, post_compute=False).label(name)
 
 
+@dispatch(Concat, (Select, Selectable), (Select, Selectable))
+def compute_up(t, lhs, rhs, **kwargs):
+    if t.axis != 0:
+        raise NotImplementedError(
+            'Cannot concat along a non-zero axis in sql',
```

Comment:
I think will point people to merge here, because this seems like it might be a common mistake. I also will switch to a valueerror. I think there needs to be some notion of a `NotImplementedOnPurpose` exception in the stdlib for cases like this.